radix-engine/src/system/bootstrap.rs

```diff
@@ -171,7 +174,7 @@ where
                 data_ingestion_receipts.push(receipt);
             }
 
-            let genesis_wrap_up_receipt = self.execute_genesis_wrap_up(next_nonce);
+            let genesis_wrap_up_receipt = self.execute_genesis_wrap_up(next_nonce, initial_time_ms);
```

Comment:
Can we do this when the clock is created - so in the first transaction - probably in the Clock constructor?

One of the main reasons I want this is to avoid the Clock ever being set to 0, because that results in annoying logic in the Gateway DB when doing SQL queries 😆

Reply:
Done